### DISCUSSION TITLE
无法使用siri展开对话

### DISCUSSION THREAD

**Discussion by vangiado** [Yue-Yang/ChatGPT-Siri#84] [Q&A] [answered]
很奇怪的我确认openai账户正常，网络情况也正常。已经换了好几个版本的捷径，都是一样的问题：
点击捷径手动输入是可以的正常对话的，用siri进入“智能聊天”也ok，但是继续往下说，就回到siri本身来去网页搜索了。  
好绝望( ；´Д｀)

**Comment by Yue-Yang** [comment, marked as answer]
和Siri对谈之后，切记不要点击中间的Siri图标，你只需要等他说话听提示，你继续说就好

**Reply by vangiado** [reply]
是这样的。。。我没有进行任何操作 但是接下来还是siri接管不是gpt
